src/presenter/trip-event-presenter.js

```diff
@@ -1,67 +1,60 @@
-import { render, replace } from '../framework/render.js';
-import ListEvent from '../view/event-list-view.js';
-import EventsItemView from '../view/events-item-view.js';
-import EventEdit from '../view/event-edit.js';
+import { render } from '../framework/render.js';
 import ListEmpty from '../view/list-empty.js';
-export default class EventPresenter {
+import PointPresenter from './point-presenter.js';
+import EventsListView from '../view/events-list-view.js';
 
-  #eventComponent = new ListEvent();
+export default class EventPresenter {
+  #eventsListView = new EventsListView();
   #points = null;
   #listContainer = null;
   #point = null;
   #offer = null;
   #destination = null;
 
+  #pointPresenter = null;
+
   constructor({listContainer, pointsModel, offersModel, destinationsModel}){
     this.#listContainer = listContainer;
     this.#point = pointsModel;
     this.#offer = offersModel;
     this.#destination = destinationsModel;
-    this.#points = pointsModel.get();
+    this.#points = new Map();
   }
 
-  #renderPoint = (point) => {
-    const pointComponent = new EventsItemView({
-      point,
-      pointDestination : this.#destination.getById(point.destination),
-      pointOffers: this.#offer.getByType(point.type)
-    });
-    // render(pointComponent,this.#eventComponent.element);
 
-    const pointEditComponent = new EventEdit({
-      point,
-      pointDestination : this.#destination.get(),
-      pointOffers: this.#offer.get()
+  #renderPoint(point){
```

Comment:
Данные точки передаем теперь в init методе, чтобы иметь воозмоность вызывать этот инит любое количество раз с новыми данными и не пересоздавать для этого каждый раз новый презентер, можно было бы оставть оба варианта: ив  конструкторе и в методе, но это избыточно. так же мы передаем onDataChange свойство, чтобы отслеживать любые изменния данных, в том числе и изменение isFavorite в них
```suggestion
#renderPoint(point) {
    const pointPresenter = new PointPresenter({
      eventsListView: this.#eventsListView,
      offersModel: this.#offersModel,
      destinationsModel: this.#destinationsModel,
      onDataChange: this.#handlePointChange
    });

    pointPresenter.init(point);

    pointPresenter.setOpenEditModeHandler(() => {
      this.#points.forEach((presenter, pointId) => {
        if (point.id !== pointId) {
          presenter.closeEditMode();
        }
      });
    });

    this.#points.set(point.id, pointPresenter);
  }
```